app/components/ScorecardProgress/ScorecardProgressShareButton.js

```diff
@@ -23,8 +25,8 @@ class ScorecardProgressShareButton extends Component {
 
   render() {
     return (
-      <TouchableOpacity onPress={() => this.shareSubmittedScorecard()}>
-        <MaterialIcon name="share" size={22} color="white" />
+      <TouchableOpacity onPress={() => this.shareSubmittedScorecard()} disabled={ isScorecardInReview(this.props.scorecard) ? true : false }>
```

Comment:
@limkimsan the ScorecardUtil.isScorecardInReview(this.props.scorecard) already return true/false, why do we need to check iif here?